gradebookng/api/src/main/java/org/sakaiproject/grading/api/Assignment.java

```diff
@@ -140,6 +140,9 @@ public class Assignment implements Serializable, Comparable<Assignment> {
     @Getter
     @Setter
     private String gradebookId;
+    @Getter
+    @Setter
+    private String gradebookUid;
```

Comment:
so the gradebookId is the same as the site which is by default a UUID, how does gradebookUid differ?

Reply:
No, gradebookId is a existing database sequence. It's used some times for direct access, this is just a setter/getter that was missing for some specific use but we're just setting a different value, not adding anything.